lib/reflectAll.js

```diff
@@ -1,5 +1,7 @@
 import reflect from './reflect';
 import isArray from 'lodash/isArray';
+import map from 'lodash/map';
```

Comment:
@andalm see https://github.com/caolan/async/blob/4931114cd6844b3d7c5e7e9d4e6d3b54d0ad95ab/lib/internal/filter.js#L1 for reference